content/automate/ManageIQ/Service/Provisioning/Email.class/__methods__/servicetemplateprovisionrequest_denied.rb

```diff
@@ -21,7 +21,9 @@ def reason
 end
 
 def approver_href(appliance)
-  " <a href='https://#{appliance}/miq_request/show/#{@miq_request.id}'</a>"
+  body = "<a href='https://#{appliance}/miq_request/show/#{@miq_request.id}'"
+  body += ">https://#{appliance}/miq_request/show/#{@miq_request.id}</a>"
+  body
```

Comment:
@jeffmcutter You can avoid introducing the `body` variable by using the `\` symbol at the end of the first line to concatenate them together.

```ruby
def approver_href(appliance)
  "<a href='https://#{appliance}/miq_request/show/#{@miq_request.id}'" \
  ">https://#{appliance}/miq_request/show/#{@miq_request.id}</a>"
end
```

Also, the original string started with a blank space.  I am thinking that is still needed, but would make more sense if this method returned the string without the space and the caller adds it where needed.

That would mean moving the leading space character to the `approver_text ` method here: https://github.com/ManageIQ/manageiq-content/blob/bac532060d47a72cc8723e3d7c0a19c12c6bda16/content/automate/ManageIQ/Service/Provisioning/Email.class/__methods__/servicetemplateprovisionrequest_denied.rb#L36-L37

Reply:
I copied it from https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/Service/Provisioning/Email.class/__methods__/servicetemplateprovisionrequest_pending.rb#L22 in the spirit of staying consistent with the existing code.  So if you're going to fix it here, I think you should fix it there also.